packages/wonder-blocks-tokens/src/theme/semantic/semantic-color.ts

```diff
@@ -817,5 +817,16 @@ export const semanticColor = {
                 text: core.foreground.critical.strong,
             },
         },
+        shared: {
+            strong: {
+                background: core.background.neutral.strong,
+                border: core.border.neutral.strong,
+                text: core.foreground.inverse.strong,
+            },
+            actions: {
+                subdued: core.foreground.instructive.default,
+                strong: core.foreground.inverse.strong,
+            },
```

Comment:
From what I understand, the `actions` tokens are used for [the links in feedback components ](https://www.figma.com/design/EuFu0U7gqc1koXm8ZhlOLp/%E2%9A%A1%EF%B8%8F-Components?node-id=6875-2059&m=dev) (Banners and Toasts). I think it makes sense to keep `actions` in the `shared` category!